src/modules/45-dashboards/components/DashboardCard/__tests__/DashboardCard.test.tsx

```diff
@@ -10,11 +10,29 @@ import { act, fireEvent, render, RenderResult, screen, within } from '@testing-l
 import { TestWrapper } from '@common/utils/testUtils'
 import type { DashboardModel } from 'services/custom-dashboards'
 import { DashboardType } from '@dashboards/types/DashboardTypes.types'
+import { DashboardsContextProvider } from '@dashboards/pages/DashboardsContext'
+import { FolderType } from '@dashboards/constants/FolderType'
 import type { StringKeys } from 'framework/strings'
+import * as customDashboardServices from 'services/custom-dashboards'
 import DashboardCard, { DashboardCardProps } from '../DashboardCard'
 
 const testTitle = 'test title'
 
+const mockFolderOne: customDashboardServices.FolderModel = {
+  id: '1',
+  name: 'testName',
+  title: 'testTitle',
+  type: FolderType.ACCOUNT,
+  child_count: 0,
+  created_at: '01/01/2022'
+}
+
+const mockGetFolderResponse: customDashboardServices.GetFoldersResponse = {
+  resource: [mockFolderOne],
+  items: 0,
+  pages: 0
```

Comment:
Should these not both be `1` as you do have results?
```suggestion
  items: 1,
  pages: 1
```